src/scripts/check-links.mjs

```diff
@@ -108,6 +108,16 @@ for (const filePath of filePaths) {
                 location
             )
         }
+
+        // Check anchors
+
+        if (link.includes('#')) {
+            const relativeTarget = new URL(link, fileUrl.href)
+            if (relativeTarget.href.startsWith('file://')) {
+                console.log('===')
+                console.log(`from ${fileUrl.href} to ${relativeTarget.href}`)
```

Comment:
Seems to be working, need to open the file and check if the anchor is there. Will require translating the headings in the target into ids.